assets/helpers/payPalExpressCheckout/__tests__/payPalExpressReducersTest.js

```diff
@@ -11,7 +11,7 @@ describe('PayPal Reducer Tests', () => {
 
   it('should return the initial state', () => {
 
-    expect(reducer(undefined, {})).toMatchSnapshot();
+    expect(reducer(13, 'GBP')(undefined, {})).toMatchSnapshot();
```

Comment:
If you pass undefined then the default value is used.  See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters (especially "Passing undefined vs. other falsy values")